src/test/java/org/takes/misc/VerboseIteratorTest.java

```diff
@@ -0,0 +1,110 @@
+package org.takes.misc;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+import org.hamcrest.MatcherAssert;
+
+import static org.hamcrest.Matchers.equalTo;
+
+import org.hamcrest.collection.IsIterableWithSize;
+
+/**
+ * Tests for {@link VerboseIterator}.
+ * @author marcus.sanchez (sanchez.marcus@gmail.com)
+ * @version $Id$
+ * @since 0.15.1
+ */
+public class VerboseIteratorTest {
+
+    /**
+     * VerboseIterator can return next value on a valid list.
+     */
+    @Test
+    public void returnsNextValue() {
+        MatcherAssert.assertThat(
+            new VerboseIterable<String>(
+                    Arrays.asList(
```

Comment:
@m4solutions this assert looks better than the previous one, but this line is misindented. `Arrays.asList` should start 4 spaces to the right of the column where `new VerboseIterable` starts. So:
```java
    MatcherAssert.assertThat(
        new VerboseIterable<String>(
            Arrays.asList(
                "Accept: text/plain",
                "Accept-Charset: utf-8",
                "Authorization: Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==",
                "Cache-Control: no-cache",
                "From: user@example.com"
            ),
            "Empty Error Message"
        ).iterator().next(),
        equalTo("Accept: text/plain")
    );
```

See how everything aligns nicely?